### PR TITLE
Inventory develop

### PR DESCRIPTION
**Pull request by justRunnz**

### CODE REVIEW

File: client/src/components/inventory/stats/MostUsedMaterials.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+      PointElement,
+      Filler,
+      Tooltip,
+      Legend
+    );
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(
+          "http://localhost:5050/inventory/statistics2"
```
Comment (N0ku) [inline client/src/components/inventory/stats/MostUsedMaterials.js:42]: Still localhost here

File: client/src/components/inventory/stats/TotalMacCount.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import "chartjs-plugin-datalabels";
+import React, {useEffect, useState} from "react";
+
+export default function TotalMacCount() {
+  const [macCount, setMacCount] = useState(0);
+
+  useEffect(() => {
+    const fetchMacData = async () => {
+      try {
+        const response = await fetch("http://localhost:5050/inventory"); // Endpoint pour récupérer toutes les données de la collection "inventory"
```
Comment (N0ku) [inline client/src/components/inventory/stats/TotalMacCount.js:11]: localhost here too

File: client/src/components/profil/ProfilFormUpdate.js
Comment (N0ku) [inline client/src/components/profil/ProfilFormUpdate.js:1]: The classes have change, possible to refacto with the class collections now ?

File: client/src/pages/groups/Groups.js
Comment (N0ku) [inline client/src/pages/groups/Groups.js:1]: This is a very old version, possible to not merge it ? I can remerge if necessary

File: client/src/pages/inventory/InventoryPdfGenerator.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React, {useEffect, useState} from "react";
+
+function PdfGenerator() {
+  const [downloadLink, setDownloadLink] = useState("");
+
+  const generatePdf = async () => {
+    try {
+      const response = await fetch(
+        "http://localhost:5050/inventory/pdfGenerator"
```
Comment (N0ku) [inline client/src/pages/inventory/InventoryPdfGenerator.js:9]: Localhost 👀

File: client/src/pages/inventory/InventoryStatistics.js [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+      Title,
+      BarController,
+      BarElement,
+      TimeScale
+    );
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(
+          "http://localhost:5050/inventory/statistics"
```
Comment (N0ku) [inline client/src/pages/inventory/InventoryStatistics.js:39]: localhost (again 😅)

File: client/src/pages/inventory/InventoryStatistics.js [hunk 5]
```diff
@@ -0,0 +1,372 @@
...
+      PointElement,
+      Filler,
+      Tooltip,
+      Legend
+    );
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(
+          "http://localhost:5050/inventory/statistics2"
```
Comment (N0ku) [inline client/src/pages/inventory/InventoryStatistics.js:199]: Localhost

File: client/src/pages/presence/Presence.js
Comment (N0ku) [inline client/src/pages/presence/Presence.js:1]: Very old version here too, need to avoid merge it

File: server/firebase.js [hunk 6]
```diff
@@ -1,18 +1,22 @@
...
+  Timestamp,
+} = require("firebase-admin/firestore");
+const {getAuth} = require("firebase-admin/auth");
 const admin = require("firebase-admin");
 
 const serviceAccount = require("./credentials.json");
 
 const app = initializeApp({
   credential: cert(serviceAccount),
-  storageBucket: process.env.REACT_APP_STORAGEBUKET,
```
Comment (N0ku) [inline server/firebase.js:10]: the address is not good here no ?

File: server/server.js
Comment (N0ku) [inline server/server.js:1]: This file need to be formatted and it's necessary to delete useless comments code. Also need the request in the server for the others backlogs

**Review by N0ku — APPROVED** [review]

File: yarn.lock
Comment (N0ku) [inline yarn.lock:1]: Why we have this file at the root of the project ?

File: server/firebase.js (hunk 6 above)
Comment (Jerance) [reply server/firebase.js:10]: I use storage bucket in backlog ressources, can you re-fix that and readd in const app storageBucket

**Pull request closed without merging**